BrowserKit/Sources/ComponentLibrary/Headers/HeaderView.swift

```diff
@@ -15,74 +15,75 @@ public final class HeaderView: UIView, ThemeApplicable {
         static let smallFaviconImageSize: CGFloat = 20
         static let maskFaviconImageSize: CGFloat = 32
         static let horizontalMargin: CGFloat = 16
+        static let iconAndLabelSpacing: CGFloat = 12
         static let headerLabelDistance: CGFloat = 2
         static let separatorHeight: CGFloat = 1
         static let closeButtonSize: CGFloat = 30
         static let warningIconSize: CGFloat = 24
     }
-
+    
```

Comment:
Heya. I dunno what happened, but all this whitespace will fail on swiftlint, so you'll have to fix it. :)